pkg/volume/csi/csi_client.go

```diff
@@ -158,6 +173,82 @@ func (c *csiDriverClient) NodeUnpublishVolume(ctx grpctx.Context, volID string,
 	return err
 }
 
+func (c *csiDriverClient) NodeStageVolume(ctx grpctx.Context,
+	volID string,
+	publishInfo map[string]string,
+	stagingTargetPath string,
+	fsType string,
+	accessMode api.PersistentVolumeAccessMode,
+	volumeAttribs map[string]string,
+) error {
+	glog.V(4).Info(log("calling NodeStageVolume rpc [volid=%s,staging_target_path=%s]", volID, stagingTargetPath))
+	if volID == "" {
+		return errors.New("missing volume id")
+	}
+	if stagingTargetPath == "" {
+		return errors.New("missing staging target path")
+	}
+	if err := c.assertConnection(); err != nil {
+		glog.Errorf("%v: failed to assert a connection: %v", csiPluginName, err)
+		return err
+	}
+
+	req := &csipb.NodeStageVolumeRequest{
```

Comment:
Pass in secret (`node_stage_secrets` field). It is specified on the Kubernetes side in the `NodeStageSecretRef` in `CSIPersistentVolumeSource`.

Examples here:
https://github.com/kubernetes/kubernetes/pull/60118
https://github.com/kubernetes/kubernetes/pull/60382